lib/elixir/lib/keyword.ex

```diff
@@ -1316,4 +1316,87 @@ defmodule Keyword do
   def size(keywords) do
     length(keywords)
   end
+
+  @doc """
+  Filters `keywords` i.e. returns a keyword list containing
+  only the elements from `keywords` for which the function
+  `fun` returns a truthy value.
```

Comment:
```suggestion
  Filters a keyword list, i.e. returns a keyword list containing
  only the pairs for which `fun` returns a truthy value.
```

Reply:
I chose the terser wording provided by @christhekeele 's suggestion above.